src/dmd/func.d

```diff
@@ -2834,6 +2834,23 @@ FuncDeclaration resolveFuncCall(const ref Loc loc, Scope* sc, Dsymbol s,
                 tiargsBuf.peekChars(), fargsBuf.peekChars());
 
             printCandidates(loc, td);
+
+            if (fargs && fargs.length == 2)
+            {
+                MatchAccumulator m2;
+                Expressions* ia = new Expressions(2);
```

Comment:
`Expressions ia = Expressions(2);`
so memory will be released at end of the scope